### PR TITLE
chore: ignore test files in test coverage report

### PR DESCRIPTION
**Pull request by booc0mtaco**
### Summary:

This should remove the stories and other test files from the code coverage, since it's currently reporting on those. Test how ignore changes the reporting.

- also add badge for coverage

### Test Plan:

- [ ] Wrote [automated tests](https://czi.atlassian.net/wiki/x/Hbl1H)
- [ ] CI tests / new tests are not applicable
- [ ] Manually tested my changes, but I want to keep the details secret
- [x] Manually tested my changes, and here are the details:
  - test how the coverage changes by ignoring the test and story files from reporting

### CODE REVIEW

**Comment by booc0mtaco** [comment]
Verified that the coverage changed in an expected way (should remain the same overall percentage or slightly increase), and that the report shows only test files being ignored:

https://app.codecov.io/gh/chanzuckerberg/edu-design-system/commit/f2b405cda47b59a6112e3426cfc4312abbbf058b/indirect-changes

File: codecov.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+ignore:
+  - "**/*.stories.ts"
+  - "**/*.stories.tsx"
+  - "**/*.test.ts"
+  - "**/*.test.tsx"
```
Comment (timzchang) [inline codecov.yml:5]: could also do:
```
**/*.?(test|stories).ts?(x)
```

**Review by timzchang — APPROVED** [review]
lgtm!

File: codecov.yml (hunk 1 above)
Comment (booc0mtaco) [reply codecov.yml:5]: @timzchang yesssss

**Review by jeremiah-clothier — APPROVED** [review]
91% 😮

**Pull request merged**